app/cafe.py

```diff
@@ -0,0 +1,22 @@
+from datetime import date
+
+from app.errors import (
+    NotVaccinatedError,
+    NotWearingMaskError,
+    OutdatedVaccineError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if not visitor.get("vaccine"):
+            raise NotVaccinatedError("The visitor must have a vaccine")
+        if visitor.get("vaccine").get("expiration_date") < date.today():
```

Comment:
If vaccine key exists, but it's None, then you'll try to do None.get('expiration_date') that will cause an error. When you develop something like this, you should try to handler such cases because you never know what data is used in tests or what data you'll get when a real user runs your program